de.uni_heidelberg.zah.GaiaSky.yaml

```
@@ -0,0 +1,44 @@
+app-id: de.uni_heidelberg.zah.GaiaSky
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.openjdk11]
+command: gaiasky
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --env=PATH=/app/bin:/app/jre/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: [/usr/lib/sdk/openjdk11/install.sh]
+
+  - name: gaiasky
+    buildsystem: simple
+    build-commands:
+       - mkdir -p /app/bin/bookmarks/ /app/bin/conf/ /app/bin/i18n/ /app/bin/lib/ /app/bin/mappings/ /app/bin/music/ /app/bin/scripts/
+       - cp -R bookmarks/* /app/bin/bookmarks/
+       - cp -R conf/* /app/bin/conf/
+       - cp -R i18n/* /app/bin/i18n/
+       - cp -R lib/* /app/bin/lib/
+       - cp -R mappings/* /app/bin/mappings/
+       - cp -R music/* /app/bin/music/
+       - cp -R scripts/* /app/bin/scripts/
+       - rm /app/bin/lib/*-macos.jar
+       - rm /app/bin/lib/*-windows.jar
+       - install -D gs_icon.svg /app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+       - install -D gaiasky /app/bin/gaiasky
+       - install -D de.uni_heidelberg.zah.GaiaSky.desktop /app/share/applications/${FLATPAK_ID}.desktop
+       - install -D de.uni_heidelberg.zah.GaiaSky.xml /app/share/metainfo/${FLATPAK_ID}.xml
```

```suggestion
       - install -D de.uni_heidelberg.zah.GaiaSky.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
```
the build fails due to this.